src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -841,27 +874,34 @@ def convert_paint_by_example_checkpoint(checkpoint):
     return model
 
 
-def convert_open_clip_checkpoint(checkpoint):
-    text_model = CLIPTextModel.from_pretrained("stabilityai/stable-diffusion-2", subfolder="text_encoder")
+def convert_open_clip_checkpoint(checkpoint, prefix="cond_stage_model.model."):
+    # text_model = CLIPTextModel.from_pretrained("stabilityai/stable-diffusion-2", subfolder="text_encoder")
```

Comment:
Are we not affecting the SD 2 conversion process with this one?

Reply:
Need to double check!